app/app.js

```diff
@@ -24,6 +24,12 @@ const Menu = remote.Menu;
 const MenuItem = remote.MenuItem;
 
 const tinycolor = require('tinycolor2');
+const nunjucks = require('nunjucks');
```

Comment:
hrm, i'd actually prefer to keep right click available on the entire tile since the hex and value are pretty small targets